public/manifest.json

```diff
@@ -1,19 +1,81 @@
 {
     "name": "Wild Animal Suffering",
-    "short_name": "Wild Animal Suffering",
+    "short_name": "WAS.org",
+    "lang": "en-US",
+    "start_url": "/",
+    "display": "minimal-ui",
     "theme_color": "#28323a",
     "background_color": "#b7b2a0",
-    "display": "standalone",
-    "scope": "/",
-    "start_url": "/",
-    "icons": [{
-        "src": "/icons/android-chrome-192x192.png",
+    "prefer_related_applications": "false",
+    "icons": [
+      {
+        "src": "icons/logo-72.png",
+        "sizes": "72x72",
+        "type": "image/png"
+      },
+      {
+        "src": "icons/logo-96.png",
+        "sizes": "96x96",
+        "type": "image/png"
+      },
+      {
+        "src": "icons/logo-120.png",
+        "sizes": "120x120",
+        "type": "image/png"
+      },
+      {
+        "src": "icons/logo-128.png",
+        "sizes": "128x128",
+        "type": "image/png"
+      },
+      {
+        "src": "icons/logo-144.png",
+        "sizes": "144x144",
+        "type": "image/png"
+      },
+      {
+        "src": "icons/logo-152.png",
+        "sizes": "152x152",
+        "type": "image/png"
+      },
+      {
+        "src": "icons/logo-180.png",
+        "sizes": "180x180",
+        "type": "image/png"
+      },
+      {
+        "src": "icons/logo-192.png",
         "sizes": "192x192",
         "type": "image/png"
-    }, {
-        "src": "/icons/android-chrome-512x512.png",
-        "sizes": "512x512",
+      },
+      {
+        "src": "icons/logo-256.png",
+        "sizes": "256x256",
+        "type": "image/png"
+      },
+      {
+        "src": "icons/logo-384.png",
+        "sizes": "384x384",
         "type": "image/png"
-
-    }]
-}
\ No newline at end of file
+      },
+      {
+        "src": "icons/logo-512.png",
+        "sizes": "512x512",
+        "type": "image/png",
+        "purpose": "any"
+      },
+      {
+        "src": "icons/logo-640.png",
+        "sizes": "640x640",
+        "type": "image/png",
+        "purpose": "any maskable"
+      },
+      {
+        "src": "icons/logo-1024.png",
+        "sizes": "1024x1024",
+        "type": "image/png",
+        "purpose": "any maskable"
+      }
+    ]
+  }
+  
```

Comment:
Re-save this file after installing the Editorconfig plugin in VS Code to get rid of this warning.

Reply:
@GRardB It didn't change anything, I even reinstalled and relaunched. Maybe we can look at it on vikings. In either case I think I solved it manually?